temper-syntax/temper-pygments/temper-pygments.temper.md

```diff
@@ -60,15 +60,15 @@ Main thing, though, is the list of rules for definition tokens.
           new Rule(raw"\d+\.?\d*|\.\d+", Kind.number),
           new Rule("@${nameRegex}", Kind.nameDecorator),
           new Rule(nameRegex, Kind.nameKind),
-        ].as<List<RuleOption>>()),
+        ].as<List<RuleOption>>() orelse panic()),
```

Comment:
These casts are just for better type inference on the List type parameter. Ideally, we can improve in the future such that we need neither the cast nor the panic.